Add Ethereum Foundation page (#989)
ethereum/ethereum-org-website: docs/foundation/index.md

---
title: Ethereum Foundation
meta:
  - name: description
    content: Learn about the Ethereum Foundation (EF), a non-profit organization dedicated to supporting Ethereum and related technologies.
  - property: og:title
    content: Ethereum Foundation | Ethereum.org
  - property: og:description
    content: Learn about the Ethereum Foundation (EF), a non-profit organization dedicated to supporting Ethereum and related technologies.
sidebar: false
lang: en-US
---

# About the Ethereum Foundation

<div>
  <img :src="$withBase('/ef-logo.png')" class="ef-logo" />
</div>

The [Ethereum Foundation](http://ethereum.foundation/) (EF) is a non-profit organization dedicated to supporting [Ethereum](/what-is-ethereum/) and related technologies.

The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor are they the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).

> Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.

Reviewer: ```suggestion
```

Let's move this bit to the very bottom of the page, in order to suggest to the reader that there are more EF initiatives than just the 2 listed here

> ## Initiatives of the Ethereum Foundation

Author: ```suggestion
## Ethereum Foundation Initiatives
```

### Ecosystem Support Program

The [Ecosystem Support Program](https://ecosystem.support/) exists to provide both financial and non-financial support to projects and entities within the greater Ethereum community, in order to accelerate the growth of the ecosystem. The Ecosystem Support Program is an expansion of the original Ethereum Grants Program which mainly focused on financial support.

Learn more about the Ecosystem Support Program, past grant recipients, and the grant application process at [ecosystem.support](https://ecosystem.support/). You can also view the [Ecosystem Support Program Blog](https://blog.ethereum.org/category/ecosystem-support-program/) for their latest news and announcements.

### Devcon

Since 2014, the Ethereum Foundation has organized Devcon, the annual conference for all Ethereum developers, researchers, thinkers, and makers.

You can access video content of conference presentations for every year since its inception at [archive.devcon.org](https://archive.devcon.org/).

> Learn more at [devcon.org](https://devcon.org/) or check out the [Devcon Blog](https://blog.ethereum.org/category/devcon/) for the latest announcements.

Reviewer: ```suggestion
Learn more at [devcon.org](https://devcon.org/) or check out the [Devcon Blog](https://blog.ethereum.org/category/devcon/) for the latest announcements.

Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```

### Ecosystem Support Program

The [Ecosystem Support Program](https://ecosystem.support/) exists to provide both financial and non-financial support to projects and entities within the greater Ethereum community, in order to accelerate the growth of the ecosystem. The Ecosystem Support Program is an expansion of the original Ethereum Grants Program which mainly focused on financial support.

Learn more about the Ecosystem Support Program, past grant recipients, and the grant application process at [ecosystem.support](https://ecosystem.support/). You can also view the [Ecosystem Support Program Blog](https://blog.ethereum.org/category/ecosystem-support-program/) for their latest news and announcements.

### Devcon

Since 2014, the Ethereum Foundation has organized Devcon, the annual conference for all Ethereum developers, researchers, thinkers, and makers.

You can access video content of conference presentations for every year since its inception at [archive.devcon.org](https://archive.devcon.org/).

Learn more at [devcon.org](https://devcon.org/) or check out the [Devcon Blog](https://blog.ethereum.org/category/devcon/) for the latest announcements.

> Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.

Author: ```suggestion
For more on the Foundation and their work, visit [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```
